libvmi/libvmi.h

```diff
@@ -661,7 +667,7 @@ status_t vmi_init(
     vmi_mode_t mode,
     void* domain,
     uint32_t init_flags,
-    void *init_data,
+    vmi_init_data_t *init_data,
```

Comment:
Looks to me like it would be fine to keep this as a `void*` in the API. My reasoning with having this as a generic pointer was that the drivers can cast it to whatever type they need it as, and as long as its not a complex structure that needs to be passed in, there is no need for a struct definition in the public header.